src/test/rustdoc-js-std/parser-paths.js

```diff
@@ -0,0 +1,75 @@
+const QUERY = ['A::B', 'A::B,C',  'A::B<f>,C'];
```

Comment:
Let's add another test case here. It's one that I had to fix in rustdoc-search-ref (search.js got it right the first time).

    mod::a

An old version of rustdoc-search-ref parsed this as a `mod:` filter followed by an invalid path, and gave an error. Because the type filters that rustdoc supports *aren't* all keywords, we need to parse this as a path.